pyweek36/main.py

```diff
@@ -195,9 +199,17 @@ def on_update(self, delta_time):
         # Move items in the physics engine
         self.physics_engine.step()
 
+        self.camera.move_to(
+            (
+                max(self.player_sprite.center_x - self.camera.viewport_width / 2, 0),
+                max(self.player_sprite.center_y - self.camera.viewport_height / 2, 0),
```

Comment:
The current camera movement is quite sharp, I think adding some damping could help

We could also add a deadzone so the camera doesn't move every time you move/jump, and lookahead on the x axis